sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Release History
 
-## 7.3.0-beta.2 (Unreleased)
+## 7.2.1 (2021-05-12)
+### Fixed
+- Fix the issue where 'ServiceBusProcessorClient:maxConcurrentCalls' will result in more messages being cached in SDK  
+  and message lock expire.
```

Comment:
This doesn't really make sense with the following line, was the issue a case where messages would be cached even after the message lock expired?